aztk/spark/client/cluster/helpers/list.py

```diff
@@ -0,0 +1,13 @@
+import azure.batch.models.batch_error as batch_error
+
+from aztk import error
+from aztk.spark import models
+from aztk.utils import helpers
+
+
+def list_clusters(core_cluster_operations):
+    try:
+        software_metadata_key = "spark"
```

Comment:
isn't that a constant?

Reply:
yea -- looks like a couple places that needed to change to `models.Software.spark`